package.json

```diff
@@ -36,6 +36,8 @@
   "devDependencies": {
     "@babel/plugin-transform-typescript": "^7.5.5",
     "@ember/optional-features": "^0.7.0",
+    "@ember/ordered-set": "^2.0.3",
+    "@glimmer/env": "^0.1.7",
```

Comment:
glimer/env and ordered-set being added here seems suspect

Reply:
They are being required in one of the packages, and I hoisted all of the dependencies to the main package.json. See comment below.